### PR TITLE
Jsonstat take 2 

### PR DESCRIPTION
**Pull request by JohannesFinsveen**

### CODE REVIEW

**Comment by PerIngeVaaje** [comment]
Looks ok to me

File: PxAPI-2.yml [hunk 1]
```diff
@@ -802,6 +806,257 @@ components:
...
+          type: object
+          additionalProperties:
+            type: object
+            properties:
+              base:
+                type: string
+              decimals: 
+                type: integer
+      additionalProperties: false
+    filters:
```
Comment (likp) [inline PxAPI-2.yml:971]: In the custom JSON we changed filters to codelists and do not distinct between vs and agg do we whant to do the same for JSON-STAT2?

Comment (JohannesFinsveen) [reply PxAPI-2.yml:971]: Thanks, I think the jsonstat example used was made when codelists were filters. There are some other changes  to Dataset coming ....

Comment (oyessb) [reply PxAPI-2.yml]: Changed filters to codelists. Moved note for table to extension to support mandatory notes. refperiod  moved to extension for dimension.

Ready for review :)

**Review by likp — APPROVED** [review]
Looks good

**Pull request merged**